docs/spec/governance.rst

```diff
@@ -6,124 +6,109 @@ Governance
-
-
-Parameters (Council)
-~~~~~~~~~~~~~~~~~~~~
-
-
-MaxProposals
-------------
-
-The maximum number of proposals allowed in the queue.
-
-.. note:: **Default Value**: 100
-
-
-MaxMembers
-----------
-
-The maximum number of participants allowed in the council.
-
-.. note:: **Default Value**: 100
+- **Optimistic Governance**
+    - No **Council**, only public proposals from community
+    - Community can elect a **Technical Committee** to fast-track proposals
+    - Referenda are Super-Majority Against (Negative Turnout Bias) by default
 
+- **Stake-To-Vote**
+    - Adopted from Curve's governance model
+    - Users lock the native governance token
+    - Longer lockups give more voting power
 
-Parameters (Voting)
-~~~~~~~~~~~~~~~~~~~
-
-
-EnactmentPeriod
----------------
-
-The period to wait before any approved change is enforced.
-
-.. note:: **Default Value**: 1 Day
-
-
-LaunchPeriod
-------------
-
-The interval after which to process a new referenda from the queue.
-
-.. note:: **Default Value**: 2 Days
+.. figure:: ../figures/spec/governance.jpeg
+    :alt: Governance Architecture
+    :scale: 30
+    :align: center
 
+An important distinction is the ``negative turnout bias`` (Super-Majority Against​) voting threshold. This is best summarized by the `Polkadot <https://wiki.polkadot.network/docs/learn-governance>`_ docs:
 
-VotingPeriod
-------------
+    A heavy super-majority of nay votes is required to reject at low turnouts, but as turnout increases towards 100%, it becomes a simple majority-carries as below.
 
-The period to allow new votes for a proposal or referenda (**Council**).
+    .. math::
 
-.. note:: **Default Value**: 2 Days
+        \frac{ against }{ \sqrt{ electorate } } < \frac{ approve }{ \sqrt{ turnout } }
 
+Terminology
+~~~~~~~~~~~
 
-FastTrackVotingPeriod
----------------------
+- **Proposals** are community-supported motions to perform system-level actions.
+- **Referenda** are accepted proposals undergoing voting.
 
-The period to allow new votes for a proposal or referenda (**Technical Committee**).
+Processes
+~~~~~~~~~
 
-.. note:: **Default Value**: 3 Hours
+Proposals
+---------
 
+1. Account submits public proposal with deposit (``> MinimumDeposit``)
+2. Account "seconds" proposal with additional deposit
+3. New referenda are started every ``LaunchPeriod``
+4. Community can vote on referenda for the ``VotingPeriod``
+5. Votes are tallied after ``VotingPeriod`` expires
+6. System update executed after ``EnactmentPeriod``
+7. Token voters can unlock balance after ``end + EnactmentPeriod * conviction``
 
-CooloffPeriod
--------------
+Technical Committee
+-------------------
 
-The period that a vetoed proposal may not be re-submitted.
+1. Community creates proposal as above
+2. TC may fast track before ``LaunchPeriod``
+3. The new referendum is started immediately
+4. Community can vote on referenda for the ``FastTrackVotingPeriod``
 
-.. note:: **Default Value**: 7 Days
+**Cancellation**
```

Comment:
Note: this is equivalent to a veto. Was not part of previous discussions - but may be useful and there is an ongoing discussion if we want to keep this. 
Suggestion: comment out for now and re-add when we finalize the decision on TC veto vs no veto. 

Attention: if we allow TC veto, this MUST NOT be applicable to election of a new TC (otherwise, the TC can prevent being removed by community which would centralize the system)